src/lib/Http/FormatAttributeHtmlFormatRequestMatcher.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Http;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher;
+
```

Comment:
This class name is slightly confusing it sounds like a combination of a method and a class name